components/bme680/bme680.c

```diff
@@ -724,9 +724,8 @@ esp_err_t bme680_get_measurement_duration(const bme680_t *dev, uint32_t *duratio
     // some ms tolerance
     *duration += 5;
 
-    // ceil to next integer value that is divisible by portTICK_PERIOD_MS and
-    // compute RTOS ticks (1 ... portTICK_PERIOD_MS =  1 tick)
-    *duration = (*duration + portTICK_PERIOD_MS - 1) / portTICK_PERIOD_MS;
```

Comment:
@UncleRus if i understand it correctly, the above comment can be ignored. am I right?

Reply:
sorry, i'll create another PR for that as i found another deprecation warning.